fluent/src/parser.js

```diff
@@ -619,19 +619,18 @@ class RuntimeParser {
   getCallArgs() {
     const args = [];
 
-    if (this._source[this._index] === ')') {
-      return args;
-    }
-
     while (this._index < this._length) {
       this.skipInlineWS();
 
+      if (this._source[this._index] === ')') {
+        return args;
+      }
```

Comment:
Moving this here allows the last argument to be followed by a comma. I filed https://github.com/projectfluent/fluent/pull/84 to allow this in the spec.